phpstan.dist.neon

```diff
@@ -25,11 +29,5 @@ parameters:
             message: '#Call to static method .+ with .+ will always evaluate to true.$#'
             path: tests
             count: 19
-
-        # baseline, should only shrink
```

Comment:
should we move other exceptions to baseline file too?

Reply:
Not really, regexes above looks rather like fixed rules we want to ignore rather than something we want to fix.

Reply:
are you sure? why?
i bet it can be the case sometimes, but rather not always.
example: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6879

Reply:
Not a problem for me, I can dump everything, but instead of 37 errors in baseline we will have 1153 😉.

But seriously, I would go with separate PRs for these rules and clean it one by one.

Reply:
fair!